.github/workflows/create-kustomize-with-bratiska-cli-inhouse.yml

```diff
@@ -0,0 +1,83 @@
+name: General workflow which creates kustomize using bratiska-cli.
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'Define the type of machine to run the job on'
+        type: string
+        required: false
+        default: '["self-hosted", "Linux", "X64", "bratislava"]'
+      directory:
+        description: 'A Folder within the repository where things should be build.'
+        default: '/'
+        required: false
+        type: string
+      debug:
+        description: 'Debug flag'
+        default: ''
+        required: false
+        type: string
+      version:
+        description: 'Bratiska-cli version'
+        default: 'stable'
+        required: false
+        type: string
+
+jobs:
+  create-kustomize-with-bratiska-cli:
+    name: Create kustomize with bratiska-cli
+    runs-on: ${{fromJSON(inputs.runs-on)}}
+    defaults:
+      run:
+        working-directory: ${{ inputs.directory }}
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.3.0"
+
+      - name: Directory check
+        run: pwd
```

Comment:
Is this some kind of debug option? Or why is it needed to print working directory inside a running pipeline?